pages/_app.tsx

```diff
@@ -8,15 +8,27 @@ import { FirebaseProvider } from "@packages/features/firebase-context";
 import { withProviders } from "@packages/utils/react";
 
 import type { AppProps } from "next/app";
+import { Suspense } from "react";
+import { useIsClient } from "@packages/hooks/useIsClient";
 
 function MyApp({ Component, pageProps }: AppProps) {
-	return withProviders(
-		<Layout>
-			<CSSReset />
-			<Metadata />
-			<Component {...pageProps} />
-		</Layout>,
-		providers,
+	const isClient = useIsClient();
```

Comment:
This is too queiiizy. Can we refactor that and use correct server/client side component composition to avoid this types of checks?

We can schedule a task for that.

Reply:
@frattezi  We can, but it'd require some refactoring. I'm planning to move to tailwind + daisy-ui in an upcoming PR

Reply:
Agreed